gradle.properties

```diff
@@ -1 +1 @@
-org.gradle.jvmargs=-Xmx2560M
+org.gradle.jvmargs=-Xmx5120M
```

Comment:
No, there should be no need for this?

Reply:
The build on github failed when it was at the old value. why? I'm not sure.